docs/content/components/content/section-message/section-message.mdx

```diff
@@ -1,56 +1,162 @@
 ---
 title: SectionMessage
-caption: Display a short message with important informations.
+caption: Display important informations in a section of a screen.
+badge: updated
 ---
 
-The `<SectionMessage>` component is used to capture the users atention to provide information. It is meant to show only very relevant information. The title should contain the most relevant information about the message.
+The `<SectionMessage>` component is a block-level element designed to alert users about specific content in a designated section of the screen. It is positioned close to the relevant content to clearly indicate its connection. Section messages provide contextual feedback within a section of the page and are persistent, non-modal elements.
 
-This component should not be added on a dynymic way.
+## Anatomy
 
-There are currently three different variants of the `<SectionMessage>` component. The `info` variant is set as default. Inside the `<SectionMessage>` it lends itself to using the [`<Text>`](/componentss/text/) component.
+The section message is build within a container which includes an icon (only in b2b theme) to support the message and a title. The actual content of section messages is located under the title.
 
-## Import
+Optionally, you can add a button to dismiss the message.
 
-To import the component you just have to use this code below.
-
-```tsx
-import { SectionMessage } from '@marigold/components';
-```
+<Image
+  src="/sectionmessage/sectionmessage-anatomy.jpg"
+  alt="Anatomy of a SectionMessage"
+  width={800}
+  height={550}
+  className="mx-auto block"
+/>
 
 ## Appearance
 
+<AppearanceDemo component={title} />
+
 <AppearanceTable component={title} />
 
+## Usage
+
+Section messages are ideal for displaying important feedback related to a specific section of the page.
+
+Unlike modal dialogs, which interrupt the user's workflow, section messages remain visible without blocking interaction with other parts of the interface.
+
+<Do>
+  <Do.Description>
+    <ul>
```

Comment:
Did you not use list here because it does not have any list indicators (or whatever the dot before each item is called?)

Reply:
what do you mean? We always just list if there are more points for a do?